src/screens/choice-exercises/components/SingleChoiceExercise.tsx

```diff
@@ -49,32 +49,20 @@ const ChoiceExerciseScreen = ({ documents, documentToAnswers, onExerciseFinished
   const count = documents.length
 
   const isAnswerEqual = (answer1: Answer, answer2: Answer): boolean => {
-    return answer1.article === answer2.article && answer1.word === answer2.word
-  }
-
-  const isCorrectAlternative = (selectedAnswer: Answer): boolean => {
-    let found = false
-    currentDocument.alternatives.forEach(alternative => {
-      if (alternative.article === selectedAnswer.article && alternative.alt_word === selectedAnswer.word) {
-        setCorrectAnswer(selectedAnswer)
-        found = true
-      }
-    })
-    return found
+    return answer1.article.id === answer2.article.id && answer1.word === answer2.word
   }
 
   const correctAlternatives: Answer[] = currentDocument.alternatives.map(it => ({
-    article: it.article,
+    article: ARTICLES[it.article],
```

Comment:
Sometimes you actually compare the id, sometimes you just access the array with the id as index, I guess we should just use one of both.

Reply:
yeah, this is my quickfix for the alternative-words, where the mapping from id to Article is not jet implemented. I created an issue for it.